contracts/security/ReentrancyGuard.sol

```diff
@@ -66,4 +66,11 @@ abstract contract ReentrancyGuard {
         // https://eips.ethereum.org/EIPS/eip-2200)
         _status = _NOT_ENTERED;
     }
+
+    /**
+     * expose _status as internal function, the caller decides what to do with it
+     */
+    function _guardStatus() internal view returns (uint256) {
```

Comment:
Let's call this `_reentrancyGuardStatus`?

Reply:
Done.